### DISCUSSION TITLE
OAuth access token API Github error NotFound

### DISCUSSION THREAD

**Discussion by caiosamarone** [community/community#69336] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I'm using github API for authroziation on my webapp, but when I call (POST) "https://github.com/login/oauth/access_token" I get Not Found 404 error.

By the way, client_id, client_secret and code are not the problem, I checked. 

Any sugestions? 
Thanks.

**Comment by saifsmailbox98** [comment]
@caiosamarone Did you get it to work?

**Comment by mikeaustin** [comment]
It's got to be something simple, I just don't know what. I set Content-Type and Accept headers to "application/json", and pass client_id, client_secret, and code, but still get 404.

**Reply by mikeaustin** [reply, marked as answer]
I did get it working after I added Content-Type and Accept headers, and also removed the authorized app every time I test.

Example:

```
  const response = await fetch('https://github.com/login/oauth/access_token', {
    method: 'POST',
    headers: {
      'Content-Type': 'application/json',
      'Accept': 'application/json',
    },
    body: JSON.stringify({
      client_id: '$$$',
      client_secret: '$$$$$$$$$$',
      code,
    }),
  });
```

**Reply by kolserdav** [reply]
I did so, but then that request returns:
```
TypeError: Body is unusable
```

**Comment by mndemirag** [comment]
Hi, I'm still having this issue after adding the headers even, any other suggestions?

**Reply by AaronConlon** [reply]
Holy.It's work after I add the content type...

**Comment by ariefsn** [comment]
Thanks,
Adding `Content-Type` solved the issue.
